apps/v4/content/docs/components/marquee.md

```
@@ -0,0 +1,163 @@
+description: An infinite scrolling component that can be used to display text, images, or cards.
+component: true
+---
+
+::component-preview
+---
+name: MarqueeDemo
+description: A marquee with review cards.
+---
+::
+
+## Installation
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)